### ISSUE TITLE
formWidget : Per currency value

### ISSUE THREAD

**Issue by Alex360hd** [responsiv/currency-plugin#19]
Hello,

I'm testing this module and it does the job pretty well when dealing with currencies, congratulation to the devs !

But there is an important feature that is missing to me : **The possibility to define a value per currency**.

I noticed the "currency" formWidget that allow to define a value for the actual default currency, but then, you can only rely on currency converters in order to get the price in another currency. You often need to get a better control over your price on different currency, so, adding this possibility in that plugin would be really useful.

Basically, I imagine a "currencies" formWidget that would behave exactly like translatable fields (provided by the [translate plugin](https://octobercms.com/plugin/rainlab-translate))

See the attached screenshot (taken from the translate plugin page), you just need to replace languages by currency.

Is per currency value in the scope of this module or do I have to create my own to handle that ?

Best regards,

Alex
